ansible/roles/couchdb2-preinstall/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: Install ndg-httpsclient so we can download couch
+  pip: name="{{ item }}"
+  sudo: yes
+  with_items:
+    - ndg-httpsclient
+
+- name: Create couchdb2 dirs
+  sudo: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: 0755
+    owner: "{{ couchdb_user }}"
+    group: "{{ couchdb_group }}"
```

Comment:
Now that i think of it, this might not work because this user & group probably won't be created by this point on a clean install. @emord not sure if we want to move some of this into your ansible-galaxy package

Reply:
I don't think it matters if the parent_srcs_dir is owned by the couchdb user.

It makes sense to move teh data directory creation to that. I think that stopped working after I made this change https://github.com/andrewrothstein/ansible-couchdb-cluster/commit/826df6010e191f17c35be598e9e4ba79d538c47f

Reply:
thanks for the tip. I'll PR a change to explicitly create that dir to https://github.com/andrewrothstein/ansible-couchdb-cluster then, since it was relying on lazy creation before.